tests/test_serialised_model.py

```diff
@@ -19,27 +19,35 @@ class CustomCollection(SerialisedModelCollection):
 
     assert str(e.value) == (
         "Can't instantiate abstract class SerialisedModel with abstract methods ALLOWED_PROPERTIES"
+    ) or str(e.value) == (
+        "Can't instantiate abstract class SerialisedModel with abstract method ALLOWED_PROPERTIES"
     )
 
     with pytest.raises(TypeError) as e:
         Custom()
 
     assert str(e.value) == (
         "Can't instantiate abstract class Custom with abstract methods ALLOWED_PROPERTIES"
+    ) or str(e.value) == (
```

Comment:
Could we just check if the first part of the message is in `value`? I'm not sure what value we get from ensuring the entire message is just-so.

If we do keep both, we should have a comment on the new alternative to clarify it's for 3.9+, as it's unclear why we have both without looking at the commit.

Reply:
There’s value in the second part of the message as well, i.e. that it’s referencing the right property. I‘ve added comments in c08f315 which make it clear what can be removed when, while keeping whole assertions.